src/deploy/functions/checkIam.ts

```diff
@@ -148,7 +148,7 @@ export function mergeBindings(policy: iam.Policy, allRequiredBindings: iam.Bindi
  * @param have backend that we have currently deployed
  */
 export async function ensureServiceAgentRoles(
-  projectId: string,
+  project: { projectId: string; projectNumber: string },
```

Comment:
nit* In theory, having just `projectNumber` should be sufficient since all GCP API calls accept either projectId or projectNumber. Will leave it up to you to decide whether that would be appropriate.

Reply:
Oh snap, I didn't realize treated them as interchangeable!!